pkg/local_object_storage/engine/inhume.go

```diff
@@ -79,6 +80,18 @@ func (e *StorageEngine) inhume(prm InhumePrm) (InhumeRes, error) {
 	}
 
 	for i := range prm.addrs {
+		if !prm.forceRemoval {
+			locked, err := e.isLocked(prm.addrs[i])
+			if err != nil {
+				return InhumeRes{}, fmt.Errorf("could not check locking status: %w", err)
```

Comment:
While I agree this could be a correct behaviour, I think it is a rather big change to me merged right before the release.
Can we check what whatever is possible here and skip missing shards?
1. We do similar things for `Exists`/Tombstones in degraded mode.
2. We still check many shards, so it would be better than our current behaviour.

Reply:
so you mean just checking all the shards we can and removing the objects if all the _available_ objects allow doing that?

Reply:
Yes

Reply:
ok, added logging and shard error counter increment, check, please